tests/Pub/TestClasses/Models/UserWithBroadcastingEventsForSpecificEvents.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace PodPoint\AwsPubSub\Tests\Pub\TestClasses\Models;
+
+use PodPoint\AwsPubSub\Pub\Database\Eloquent\BroadcastsEvents;
+
+class UserWithBroadcastingEventsForSpecificEvents extends User
+{
+    use BroadcastsEvents;
+
+    public function broadcastOn($event)
+    {
+        switch ($event) {
+            case 'created':
+            case 'trashed':
+            case 'restored':
+            case 'deleted':
+                return [];
+            case 'updated':
+                return ['users'];
+        }
```

Comment:
Not sure if we should change the name of the file to reflect it is only doing Updated?

Reply:
Can do yeah.